packages/storybook/src/stories/grid-rows.stories.tsx

```diff
@@ -482,11 +481,11 @@ export function EditRowsControl() {
   }, []);
 
   const onCellEditCommit = React.useCallback(
-    (params: GridEditCellPropsParams, event?: React.SyntheticEvent) => {
+    (params: GridEditCellPropsParams, event) => {
```

Comment:
Why not doing? 
```suggestion
    (params: GridEditCellPropsParams, event?: MuiEvent<React.SyntheticEvent>) => {
```
instead of using as later down

Reply:
Fixed.